frameworks/cassandra/tests/conftest.py

```diff
@@ -0,0 +1,37 @@
+import os
+import json
+import pytest
+import shakedown
+
+
+@pytest.fixture(scope='session', autouse=True)
```

Comment:
Now that you are touching the setup functions in the test cases, this autouse isn't really appropriate as the downstream fixture can declare this one. See below.